cypress/integration/change_password.spec.js

```diff
@@ -114,37 +113,42 @@ describe('Password change flow', () => {
   context('Password exists in breach dataset', () => {
     it('displays a breached error', () => {
       cy.mockNext(200);
+      cy.intercept({
+        method: 'GET',
+        url: 'https://api.pwnedpasswords.com/range/*',
+      }).as('breachCheck');
       cy.visit(`/reset-password/fake_token`);
       cy.get('input[name="password"]').type('password');
-      // there seems to be a race condition on the change password page
-      // adding a small wait fixes the issue
-      cy.wait(300);
+      cy.wait('@breachCheck');
+      cy.contains('use a password that is hard to guess');
       cy.get('button[type="submit"]').click();
-      cy.contains('common password');
+      cy.contains('is a common password');
+      cy.get('input[name="password"]').type('iamaveryuniqueandlongstring');
+      cy.wait('@breachCheck');
+      cy.contains('At least 8 characters'); // This means success 🤷‍♂
```

Comment:
maybe checking the colour would be useful if this message also gets shown for error

Reply:
I _think_ that the next PR in this space will change this to `cy.contains('Valid email'); ` (or some similar text) so I deliberately left this vague!